### DISCUSSION TITLE
Roku client: IOTV

### DISCUSSION THREAD

**Discussion by 9123086** [iptv-org/community#485] [Show and tell]
Please find a Roku client here:
https://my.roku.com/account/add/iotv0815

**Comment by 9123086** [comment]
Manny channels which have the stream associated are not able to play.
How can help and what's the mechanism to maintain a channel is playable? 
if adding a "isPlayable" flag to each channel in the database it will be nice for client app to only show up those playable channels.

**Reply by BellezaEmporium** [reply]
It was linked to GH actions, and we are not allowed to run any tasks as such.

Is "iotv0815" using our playlists or not ? I cannot confirm it as I do not have a Roku box.

**Reply by 9123086** [reply]
Roku IOTV is using api from here:
https://iptv-org.github.io/api

how about other platform's clients? are they use the same api and the same data behind the api?
if the same data base used, the similar user experience - some channels are not playing video when user select from the UI.

**Reply by BellezaEmporium** [reply]
You are most certainly making a reference to the "streams" JSON file. It is updated daily when people change the links. Since we cannot verify if the links works or not (due to recent GH Actions limitations), we currently work with "community support".

**Reply by 9123086** [reply]
understand and thank you @BellezaEmporium  for the update.
